lib/rubocop/cop/minitest/refute_false.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `refute(actual)`
+      # instead of `assert_equal(false, actual)`.
+      #
+      # @example
+      #   # bad
+      #   assert_equal(false, actual)
```

Comment:
Shouldn't we cover `nil` as well?

Reply:
`assert_equal(nil, object)` will trigger another cop to change it to `assert_nil(object)`. Adding it here would only add confusion.